changelog.txt

```diff
@@ -0,0 +1 @@
+fixed - Stop warning about shrink-ray inclusion failure; firebase-tools is also displaying the message.
```

Comment:
Let's just call this:

    fixed - Silence warning when falling back to gzip compression